match.go

```diff
@@ -41,10 +41,7 @@ import (
 	"strings"
 )
 
-var (
-	goroot       = filepath.Clean(runtime.GOROOT())
-	gorootSrcPkg = filepath.Join(goroot, "src/pkg")
-)
+var goroot = filepath.Clean(runtime.GOROOT())
```

Comment:
Thanks. I thought `goroot` was used elsewhere, that's why I left it alone, since its value does not differ between the two versions.

However, since it's only used for `gorootSrcPkg`, it makes sense to move it there. In fact, it can be just removed.

Done in fb417b44ad915a5391a308f01e6a41117acced3d.